src/pages/PokemonDetailsPage/PokemonDetailsPage.css

```diff
@@ -105,3 +101,65 @@
 .bold {
   font-weight: bold;
 }
+
+@media screen and (max-width: 440px) {
+  .button-and-name {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  }
+
+  .copy-to-clipboard {
+    margin: 0;
+    grid-column-start: 2;
+  }
+
+  .stats-title {
+    grid-column-start: 4;
+  }
+
+  .display-section {
+    max-width: 430px;
+    flex-grow: 1;
+  }
+  .stats-window {
+    margin: 0;
+    border-radius: 0;
+    height: 662px;
+    width: 430px;
```

Comment:
In _general_ I would try to avoid ever setting a hard pixel height and width. 

I'd instead aim to let this scale up and down more fluidly as the viewport changes, you can achieve this with CSS grid, Flex etc.